tools/Google.Cloud.Tools.ReleaseManager/GenerateApisCommand.cs

```diff
@@ -181,13 +185,13 @@ void ValidateNamespaces()
 
     private void GenerateGapicApi(ApiMetadata api)
     {
-        var layout = DirectoryLayout.ForApi(api.Id, generatorOutputDirectory, generatorInputDirectory);
-        string productionDirectory = Path.Combine(layout.SourceDirectory, api.Id);
+        var apiLayout = _rootLayout.CreateApiLayout(api);
+        string productionDirectory = Path.Combine(apiLayout.SourceDirectory, api.Id);
```

Comment:
I think this is the same as a built package directory I flagged for moving into ApiLayout a few comments up. If it's here again, maybe it does make sense to move it to ApiLayout.

Reply:
Yes, exactly what I was thinking :)